Tchap/Config/BuildSettings.swift

```diff
@@ -440,4 +446,18 @@ final class BuildSettings: NSObject {
     
     // MARK: - New App Layout
     static let newAppLayoutEnabled = true
+    
+    // MARK: - QR Login
+    
+    /// Flag indicating whether the QR login enabled from login screen
+    static let qrLoginEnabledFromNotAuthenticated = true
+    /// Flag indicating whether the QR login enabled from Device Manager screen
+    static let qrLoginEnabledFromAuthenticated = false
+    /// Flag indicating whether displaying QRs enabled for the QR login screens
+    static let qrLoginEnableDisplayingQRs = false
+    
+    static let rendezvousServerBaseURL = URL(string: "https://rendezvous.lab.element.dev/")!
+
+    // MARK: - Alerts
+    static let showUnverifiedSessionsAlert = true
```

Comment:
Please add this change in Btchap and DevTchap `BuildSettings.swift` files too.

Reply:
Done